save-api/src/main/kotlin/com/saveourtool/save/api/impl/DefaultSaveCloudClient.kt

```diff
@@ -95,7 +95,7 @@ internal class DefaultSaveCloudClient(
     override suspend fun listProjects(organizationName: String): Either<SaveCloudError, List<Project>> =
             postAndCheck(
                 "/projects/by-filters",
-                requestBody = Json.encodeToString(ProjectFilters("", ORGANIZATION_NAME, ProjectStatus.values().toSet())),
+                requestBody = Json.encodeToString(ProjectFilters("", organizationName, ProjectStatus.values().toSet())),
```

Comment:
This is already fixed in #1519, so please leave `DefaultSaveCloudClient` unchanged.

Remarks on your code:

1. Since the type of `requestBody` is `Any`, and JSON serialization is already enabled for _Ktor_, you don't need to explicitly convert the value from JSON to a string. 
2. `T.values().toSet()` is actually idiomatic Java. Idiomatic Kotlin would rather use `enumValues()`:

```suggestion
                requestBody = Json.encodeToString(ProjectFilters("", organizationName, enumValues<ProjectStatus>().toSet())),
```

Reply:
Done